tools/test-static-code

```diff
@@ -37,7 +37,7 @@ if ! $PYFLAKES /dev/null >/dev/null 2>&1; then
     echo "ok 2 pyflakes test # SKIP pyflakes3 not installed"
 
 else
-    out=$($PYFLAKES test/ test/verify/check-* 2>&1 | grep -Ev "(unable to detect undefined names|defined from star imports|'parent' imported but unused|test/avocado/testvm.py)") || true
+    out=$($PYFLAKES test/ 2>&1 | grep -Ev "(unable to detect undefined names|defined from star imports|'parent' imported but unused|test/avocado/testvm.py)") || true
```

Comment:
I verified that by adding e. g. an unused `import gzip` to check-menu and running tools/test-static-code. Indeed it's still being picked up, and now it stops showing the same errors twice. Maybe that was only necessary in the Python 2 version? ISTR that it previously only recursively searched for `*.py` files, apparently it now learned to look at executables too, or look at the first line.